[
    {
        "id": 230083007,
        "name": "django_cve201919844",
        "full_name": "andripwn\/django_cve201919844",
        "owner": {
            "login": "andripwn",
            "id": 52893492,
            "avatar_url": "https:\/\/avatars.githubusercontent.com\/u\/52893492?v=4",
            "html_url": "https:\/\/github.com\/andripwn"
        },
        "html_url": "https:\/\/github.com\/andripwn\/django_cve201919844",
        "description": "PoC for CVE-2019-19844 ( https:\/\/www.djangoproject.com\/weblog\/2019\/dec\/18\/security-releases\/ )",
        "fork": false,
        "created_at": "2019-12-25T10:05:48Z",
        "updated_at": "2020-09-20T16:23:40Z",
        "pushed_at": "2020-09-20T16:23:45Z",
        "stargazers_count": 7,
        "watchers_count": 7,
        "has_discussions": false,
        "forks_count": 1,
        "allow_forking": true,
        "is_template": false,
        "web_commit_signoff_required": false,
        "topics": [],
        "visibility": "public",
        "forks": 1,
        "watchers": 7,
        "score": 0
    },
    {
        "id": 234739158,
        "name": "CVE_2019_19844",
        "full_name": "0xsha\/CVE_2019_19844",
        "owner": {
            "login": "0xsha",
            "id": 54356171,
            "avatar_url": "https:\/\/avatars.githubusercontent.com\/u\/54356171?v=4",
            "html_url": "https:\/\/github.com\/0xsha"
        },
        "html_url": "https:\/\/github.com\/0xsha\/CVE_2019_19844",
        "description": "CVE-2019-19844 Docker Edition",
        "fork": false,
        "created_at": "2020-01-18T13:32:21Z",
        "updated_at": "2022-10-17T23:08:09Z",
        "pushed_at": "2021-09-22T18:25:53Z",
        "stargazers_count": 5,
        "watchers_count": 5,
        "has_discussions": false,
        "forks_count": 3,
        "allow_forking": true,
        "is_template": false,
        "web_commit_signoff_required": false,
        "topics": [],
        "visibility": "public",
        "forks": 3,
        "watchers": 5,
        "score": 0
    }
]